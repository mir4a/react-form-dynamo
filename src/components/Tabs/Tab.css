.Tab {
  display: flex;
  flex: 1;
  flex-direction: column;
  text-align: left;
  border: 2px solid var(--btn-primary);
  padding: 15px;
}