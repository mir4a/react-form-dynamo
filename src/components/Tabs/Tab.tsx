import React, { ReactNode } from 'react';

interface Props {
  isActive?: boolean;
  children: ReactNode;
}

export default function Tab(props: Props) {
  return (
    <>
      is Active?: {JSON.stringify(props.isActive)}
      <>{props.children}</>
    </>
  );
}
