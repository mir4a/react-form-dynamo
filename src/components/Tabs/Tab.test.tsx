import React from 'react';
import renderer from 'react-test-renderer';
import Tab from './Tab';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Tab>
        <div>Content 1</div>
      </Tab>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
