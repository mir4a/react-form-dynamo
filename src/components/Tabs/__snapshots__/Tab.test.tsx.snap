// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Tab"
>
  <div>
    Content 1
  </div>
</div>
`;
